src/summon/browzine-summon-adapter.js

```diff
@@ -150,10 +150,60 @@ browzine.summon = (function() {
     return browzineEnabled;
   };
 
-  function buildTemplate(scope, browzineWebLink) {
+  function isDefaultCoverImage(coverImageUrl) {
+    var defaultCoverImage = false;
+
+    if(coverImageUrl && coverImageUrl.toLowerCase().indexOf("default") > -1) {
+      defaultCoverImage = true;
+    }
+
+    return defaultCoverImage;
+  };
+
+  function getDirectToPDFUrl(scope, data) {
+    var directToPDFUrl = null;
+
+    if(isArticle(scope)) {
+      if(data.fullTextFile) {
+        directToPDFUrl = data.fullTextFile;
+      }
+    }
+
+    return directToPDFUrl;
+  };
+
+  function showDirectToPDFLink() {
+    var enableShowDirectToPDFLink = false;
+    var config = browzine.summonArticlePDFDownloadLinkEnabled;
+
+    if(typeof config === "undefined" || config === null || config === true) {
+      enableShowDirectToPDFLink = true;
+    }
+
+    return enableShowDirectToPDFLink;
+  };
+
+  function directToPDFTemplate(directToPDFUrl) {
+    var pdfIcon = "https://s3.amazonaws.com/thirdiron-assets/images/integrations/browzine-pdf-download-icon.png";
+    var articlePDFDownloadWording = browzine.summonArticlePDFDownloadWording || "Article PDF";
```

Comment:
_non-blocking PR comment_

Could be a potential improvement in the future, but what if we collected all these default strings and put them at the top of the file in one place?  Would let John or someone else very easily see, "oh, here are all the things that a user might see in the UI," and allow easier customization that way.